ext_data/community_models/README.md

```diff
@@ -0,0 +1,11 @@
+This is an example directory where you need to add all your `.mat` or `.json` model files 
+your community consists of. 
+
+For example, you may have a community of 2 models, *Escherichia coli* and *Helicobacter pylori*, 
+in that case you could get the [`e_coli_core.json`](http://bigg.ucsd.edu/models/e_coli_core) and 
+[`iIT341.json`](http://bigg.ucsd.edu/models/iIT341) files from the [BIGG database](http://bigg.ucsd.edu/models)
+and then run `dingo`. 
+
+Remember to remove this `README.md` file if you will decide to use this directory. 
```

Comment:
Why?

Reply:
If you leave the `README.md` file, `dingo` will give you an 
`Exception: An unknown input format given to initialize a metabolic network object.` message. 

We can add a check on the code for the `README` file if you feel this is necessary.